README.md

```diff
@@ -78,6 +78,25 @@ done
 fi
 ```
 
+### Lint JS and Markdown files with Prettier on save
+
+1. Install the [Prettier VS Code Extension](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode).
+2. Run `npm install` in the docs folder you have the docs checked out in.
+3. Inside VS code, open `settings.json` and ensure the following settings are in the file:
```

Comment:
how do you get to `settings.json`? On my macbook I can only open up a GUI settings, tried searching how to do it to no avail

Reply:
updated!